demo-app/src/graphql/Subscription/hello.ts

```diff
@@ -0,0 +1,17 @@
+import sleep from '@/lib/sleep'
+import {
+  type HelloResponse,
+  type SubscriptionHelloArgs,
+} from '@/graphql/generated-types'
+
+const hello = {
+  subscribe: async function* (_: unknown, { names }: SubscriptionHelloArgs) {
```

Comment:
Oh interesting. You want to support subscriptions? At of curiosity, do you have any plans to make a subscription?

Reply:
We use subscriptions at work already for a few things, so I imagine it might be an adaption of our existing usages but within open truss.

Reply:
Yeah, I imagine there will be use cases at some point so I figured I would support it from day one if it wasn't too much trouble. That's one of the reasons I chose graphql-yoga as it comes with support and was able to get it to work with nextjs.